packages/adyen-integration/jest.config.js

```diff
@@ -9,4 +9,5 @@ module.exports = {
     },
     setupFilesAfterEnv: ['../../jest-setup.ts'],
     coverageDirectory: '../../coverage/packages/adyen-integration',
+    testPathIgnorePatterns: ['/adyenv2/e2e/', '/adyenv3/e2e/'],
```

Comment:
Do we really need this here? Thought we ignored the `/e2e` on a global level.
https://github.com/bigcommerce/checkout-js/blob/64814f14bcf585f8f52345c9b64cddf1334fa3f9/jest.preset.js#L6

Reply:
without it, adyen e2e tests are running with unit tests

Reply:
I see. Could you please move e2e tests to `packages/adyen-integration/e2e` as we put all e2e tests in this folder  of each package?